cli/cmd/plugin/standalone-cluster/main.go

```diff
@@ -12,9 +12,23 @@ import (
 )
 
 var descriptor = cliv1alpha1.PluginDescriptor{
-	Name:        "standalone-cluster",
-	Description: "Create clusters without a dedicated management cluster",
-	Group:       cliv1alpha1.RunCmdGroup,
+	Name: "standalone-cluster",
+	Description: `
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
@jpmcb can you update this deprecation notice updated to mention the use of `unmanaged-cluster` since it will ship alongside `standalone-cluster`